backend/src/apiserver/client_manager.go

```diff
@@ -214,6 +214,11 @@ func initDBClient(initConnectionTimeout time.Duration) *storage.DB {
 		glog.Fatalf("Failed to initialize the databases.")
 	}
 
+	response = db.Model(&model.Experiment{}).RemoveIndex("Name")
```

Comment:
Maybe you already verified - please make sure response.Error is nil if running this line multiple times and index already deleted, since this will run on every api server startup.

Reply:
> Maybe you already verified - please make sure response.Error is nil if running this line multiple times and index already deleted, since this will run on every api server startup.

Yes, I've verified that it's safe to call this multiple times. No error if the index is already removed.